presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -76,10 +87,12 @@ toConnectorConfigs(const protocol::SessionRepresentation& session) {
   std::unordered_map<std::string, std::unordered_map<std::string, std::string>>
       connectorConfigs;
   for (const auto& entry : session.catalogProperties) {
```

Comment:
```
  for (const auto& entry : session.catalogProperties) {
    for (const auto& property : entry.second) {
      connectorConfigs.emplace(
          toVeloxConnectorConfig(property.first), property.second);
    }
  }
```

Reply:
catalogProperties is a map of connectorId to a map of property pairs. This nested loop might not work in this case?